lib/signatures.ts

```diff
@@ -0,0 +1,18 @@
+import { promises } from 'fs';
+import { SignatureResult } from './types';
+import { getDubHashSignature } from './dubhash';
+
+const { readFile } = promises;
```

Comment:
`import { readFile } from 'fs/promises'` will work fine :)

Reply:
I think this is not available in earlier versions of node. Since we must support v10, this was the only way to import it.

Reply:
TIL!